src/test-harness/CommonTestBase.sol

```diff
@@ -44,7 +44,7 @@ contract CommonTestBase is Test {
     if (block.chainid == ChainIds.MAINNET) {
       // USDC
       if (asset == USDC_MAINNET) {
-        vm.prank(0x28C6c06298d514Db089934071355E5743bf21d60);
```

Comment:
Should we change this to a constant while we're at it?

Reply:
Yeah exactly, just getting a large address and using it, doesn't really have to be labelled